kotlin-grpc/src/test/kotlin/CoroutineServerInterceptorTest.kt

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.grpc
+
+import CoroutineServerInterceptor
+import com.linecorp.armeria.common.RequestContext
+import com.linecorp.armeria.common.grpc.GrpcStatusFunction
+import com.linecorp.armeria.internal.testing.AnticipatedException
+import com.linecorp.armeria.server.ServerBuilder
+import com.linecorp.armeria.server.ServiceRequestContext
+import com.linecorp.armeria.server.auth.Authorizer
+import com.linecorp.armeria.testing.junit5.server.ServerExtension
+import io.grpc.Metadata
+import io.grpc.ServerCall
+import io.grpc.ServerCallHandler
+import io.grpc.Status
+import org.junit.jupiter.api.extension.RegisterExtension
+import java.util.concurrent.CompletableFuture
+import java.util.concurrent.TimeUnit
+import kotlin.coroutines.suspendCoroutine
+
+internal class CoroutineServerInterceptorTest {
+    private class AuthInterceptor : CoroutineServerInterceptor {
+        private val authorizer = Authorizer { ctx: ServiceRequestContext, metadata: Metadata? ->
```

Comment:
`Metadata` is not null always.
```suggestion
        private val authorizer = Authorizer { ctx: ServiceRequestContext, metadata: Metadata ->
```

Reply:
Done in https://github.com/line/armeria/commit/47db28e48aa25a189e298960db1e4d79ba6c0915.